tests/foreman/ui_airgun/test_contentview.py

```diff
@@ -898,3 +913,346 @@ def test_positive_promote_composite_with_custom_content(session):
         result = session.contentview.promote(
             cv_composite_name, version, lce.name)
         assert 'Promoted to {}'.format(lce.name) in result['Status']
+
+
+@tier2
+def test_positive_remove_filter(session, module_org):
+    """Create empty content views filter and remove it
+
+    :id: 6c6deae7-13f1-4638-a960-d3565d93fd64
+
+    :expectedresults: content views filter removed successfully
+
+    :CaseLevel: Integration
+    """
+    filter_name = gen_string('alpha')
+    cv = entities.ContentView(organization=module_org).create()
+    with session:
+        session.contentviewfilter.create(cv.name, {
+            'name': filter_name,
+            'content_type': FILTER_CONTENT_TYPE['package'],
+            'inclusion_type': FILTER_TYPE['exclude'],
+        })
+        assert session.contentviewfilter.search(
+            cv.name, filter_name)[0]['Name'] == filter_name
+        session.contentviewfilter.delete(cv.name, filter_name)
+        assert not session.contentviewfilter.search(cv.name, filter_name)
+
+
+@tier2
+def test_positive_add_package_filter(session, module_org):
+    """Add package to content views filter
+
+    :id: 1cc8d921-92e5-4b51-8050-a7e775095f97
+
+    :expectedresults: content views filter created and selected packages can be
+        added for inclusion/exclusion
+
+    :CaseLevel: Integration
+    """
+    packages = (
+                ('cow', 'All Versions'),
+                ('bird', ('Equal To', '0.5')),
+                ('crow', ('Less Than', '0.5')),
+                ('bear', ('Range', '4.1', '4.6')))
+    filter_name = gen_string('alpha')
+    repo_name = gen_string('alpha')
+    create_sync_custom_repo(module_org.id, repo_name=repo_name)
+    repo = entities.Repository(name=repo_name).search(
+        query={'organization_id': module_org.id})[0]
+    cv = entities.ContentView(
+        organization=module_org,
+        repository=[repo]
+    ).create()
+    with session:
+        session.contentviewfilter.create(cv.name, {
+            'name': filter_name,
+            'content_type': FILTER_CONTENT_TYPE['package'],
+            'inclusion_type': FILTER_TYPE['include'],
```

Comment:
perhaps we should add this with @parametrize('inclusion_type', ['include', 'exlude']) what do you think?

Reply:
This will lead to a different scenario - if mentioned packages are excluded, then in the end we'll receive all possible packages except excluded ones, so at the very least we should implement different assertions.